docs/readme.md

```diff
@@ -0,0 +1,42 @@
+## How to use DiffusionBee
+
+This documentation is written for version 0.3.0. Parts of it may be unapplicable for other versions.
+
+### Installation
```

Comment:
This section can be simplified by removing the first step, that's pretty much known by installing all applications on macOS. I would simplify to the following:

```
### Installation

On first launch, DiffusionBee will download and install additional data for generation.
```